utils/build-presets.ini

```diff
@@ -914,20 +919,29 @@ host-test
 extra-cmake-options=-DSWIFT_ENABLE_LLD_LINKER:BOOL=OFF
 
 install-prefix=/usr
-install-llvm
```

Comment:
I just tried a full build of the latest July 24 trunk source snapshot with these flags, but with the testing flags excised, and the only problem I had was with this flag. While this works fine for the native linux build, I'm trying to avoid the cross-compiled LLVM build with the `llvm-ninja-targets-for-cross-compile-hosts=help` flag above, but this install target kicks off the full cross-compile of LLVM for Android later on when installing.

Since this preset won't create a toolchain with the Android corelibs installed in a place where the Swift compiler can detect them anyway, as mentioned below, the easiest way to fix this LLVM cross-compile is not to install LLVM either.